### ISSUE TITLE
Host live demos of JavaScript browser examples

### ISSUE THREAD

**Issue by cosinekitty** [cosinekitty/astronomy#49]
I was trying to use GitHub Pages just so I could host live demos of the JavaScript browser examples. It worked, but at the cost of having two different versions of this project with different Markdown renderers. I have turned off GitHub Pages. Now I need to create my own static website just for hosting the demos. Also need to automate the process of publishing there every time I hit a stable release.

**Comment by matheo** [comment]
I could help here with an Angular App (my speciality) to show up some cool examples on the top of the engine.
I want to learn how to render some stuff with the result data, and if I get some help from you we can get it done :)

**Comment by cosinekitty** [comment]
Hi @matheo ! I'm glad you are interested in contributing. Yes, it would be helpful to have more examples of using Astronomy Engine in some real-world applications. I don't know very much about Angular, React, etc. My goal is to keep Astronomy Engine free of any external dependencies for all the supported languages (C/C++, C#, Python, JavaScript). However, examples and demos can have external dependencies so long as they do not creep into the core libraries.

Do you have any ideas of a project you would like to build? I can definitely help with figuring out how to do various calculations.

**Comment by matheo** [comment]
> examples and demos can have external dependencies so long as they do not creep into the core libraries.
of course, the demo app should be consuming the engine from npm (but it also could consume an `astronomy.js` file as the current demos.

> Do you have any ideas of a project you would like to build?

Initially I want to find some cool 3D stuff like https://sanderblue.github.io/solar-system-threejs/ and try to model the engine data with such fancy displaying for a given `AstroTime`. In that view we could documebt the `Observers`.

Then for the `Coordinate systems` we could use another perspective from https://github.com/ofrohn/d3-celestial 

So I'm thinking about an Angular App with the `Documentation` next to cool animations, and `Demo` usages with the Moon Phases, Rise/Set, Eclipses, etc.

> I can definitely help with figuring out how to do various calculations.

Thank you very much
I will study and come back with some questions when I have them :)

**Comment by faisalr** [comment]
He @cosinekitty 

I hosted a live demo here https://faisalr.github.io/

**Comment by cosinekitty** [comment]
That looks great! Thank you for doing that. I noticed that you ask for the geographic location, but you are not using it, so the altitude and azimuth angles of the Moon are not correct for where I live. The code has a fixed latitude and longitude:

```javascript
let observer = new Astronomy.Observer(33, 33, 0);
```

**Comment by faisalr** [comment]
I fixed the demo.

**Comment by cosinekitty** [comment]
Yes, I entered my geographic coordinates and all the numbers look correct. That is great! May I add a link to it from the [browser demo page](https://github.com/cosinekitty/astronomy/tree/master/demo/browser)?

**Comment by faisalr** [comment]
Yes.

**Comment by bobhampton** [comment]
Hello @cosinekitty 

I know this is an old thread, but after seeing @faisalr live demo, I created one that will display dark sky times (times during astronomical twilight when the moon is below the horizon) for any location and date range. My partner uses it to find windows where she can take SQM light pollution measurements for a nature preserve. I'd appreciate it if you took a look if you have the time and feel free to use it on the browser demo page if you like it. Thanks for all your hard work! https://bobhampton.github.io/darkSky/

**Comment by gaxmann** [comment]
Hello,

I also like to add a link to a JS demo. It is an Astronomical Clock, which uses the path of the Sun as a dial to display true local time but also displays official time on the dial. 

https://astronomicalclock.eu/
https://astronomicalclock.eu/info/

Way to go! I love this JS astronomy engine.

**Comment by bobhampton** [comment]
@gaxmann in show and tell I asked if I should open a PR and he said he is no longer actively maintaining this repo and is looking for someone to take it over. See https://github.com/cosinekitty/astronomy/discussions/409. So we'll have to wait to get our demos added until that happens.
